src/PKSim.Assets/PKSimConstants.cs

```diff
@@ -2166,7 +2166,15 @@ public static class UI
          public static readonly string WatermarkText = "Text";
          public static readonly string WatermarkProperties = "Watermark Properties";
          public static readonly string CreateTableSolubilityAlternative = "Create as pH-Solubility table";
-         public static readonly string DoYouWantToProceedWithExportToSnapshotWithChangedSimulation = "WARNING: Some simulations are in a changed state (red icon) and may not be re-imported correctly.\nDo you wish to continue?";
+
+         public static string DoYouWantToProceed(params string[] messages) => $"WARNING:\n{messages.ToString("\n")}\n\nDo you wish to continue?";
+
+         private static readonly string _snapshotOfProjectWithChangedSimulationText = "Some simulations are in a changed state (red icon) and may not be re-imported correctly.";
```

Comment:
These are the specific messages. Please review and create another issue if they need to be updated :)